core/src/main/resources/error/error-classes.json

```diff
@@ -29,10 +41,30 @@
     "message" : [ "Invalid pivot column '%s'. Pivot columns must be comparable." ],
     "sqlState" : "42000"
   },
+  "INSERTED_VALUE_NUMBER_NOT_MATCH_FIELD_NUMBER" : {
+    "message" : [ "The number of inserted values cannot match the fields." ],
+    "sqlState" : "42000"
+  },
   "INVALID_FIELD_NAME" : {
     "message" : [ "Field name %s is invalid: %s is not a struct." ],
     "sqlState" : "42000"
   },
+  "INVALID_INSERT_INTO_CONTEXT" : {
+    "message" : [ "Invalid InsertIntoContext" ],
```

Comment:
This error message looks a big vague. What exactly the error is?

Reply:
The message "Invalid InsertIntoContext" is thrown when withInsertInto does not match.
https://github.com/apache/spark/blob/de62b5ae325741e6617e9400f78c6d0fc8cea5de/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala#L267

Reply:
This is a user-facing error message, do we expect end-users to understand this message?